src/components/KFileUpload/KFileUpload.vue

```diff
@@ -64,6 +78,62 @@
         </template>
       </KInput>
     </div>
+    <div
+      v-if="isAppearanceDropzone"
+      class="file-upload-dropzone-wrapper"
+    >
+      <div
+        ref="dropzone"
+        :aria-disabled="disabled ? 'true' : undefined"
```

Comment:
```suggestion
        :aria-disabled="disabled ? 'true' : undefined"
```
should it be `'true'` or `true`?